app/Libraries/SSO/RsaSha1.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+
+namespace App\Libraries\SSO;
+
+use Eher\OAuth\OAuthException;
+use Eher\OAuth\RsaSha1 as BaseRsaSha1;
+
+class RsaSha1 extends BaseRsaSha1
+{
+    private $cert = false;
+
+    public function __construct($cert)
+    {
+        $this->cert = $cert;
+    }
+
+    public function fetch_private_cert(&$request)
```

Comment:
Do we need to use PBR explicitly here? If `$request` is an object then it's PBR by default.

Reply:
I have no idea, this is code taken straight from the current VATSIM SSO repo we use on core, don't particularly want to change anything without have solid tests in for it (which we can't do with most SSO stuff anyway...). Happy to change if you think this won't effect it though?

Reply:
(It is also included on the parent class that is overrides)

Reply:
Yeah that's fair